cfn-resources/project/template.yml

```diff
@@ -11,18 +11,17 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       Handler: bootstrap
-      Runtime: go1.x
+      Runtime: provided.al2
       CodeUri: bin/
 
   TestEntrypoint:
     Type: AWS::Serverless::Function
     Properties:
       Handler: bootstrap
-      Runtime: go1.x
+      Runtime: provided.al2
```

Comment:
how is it in other resources? would it make a sense a follow-up PR to to the change in bulk for all resources?